pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupService.java

```diff
@@ -672,8 +690,10 @@ private void checkRGCreateParams(String rgName, org.apache.pulsar.common.policie
     private ConcurrentHashMap<String, ResourceGroup> namespaceToRGsMap = new ConcurrentHashMap<>();
 
     // Maps to maintain the usage per topic, in produce/consume directions.
-    private ConcurrentHashMap<String, BytesAndMessagesCount> topicProduceStats = new ConcurrentHashMap();
-    private ConcurrentHashMap<String, BytesAndMessagesCount> topicConsumeStats = new ConcurrentHashMap();
+    private ConcurrentHashMap<String, BytesAndMessagesCount> topicProduceStats =
+                                                new ConcurrentHashMap<String, BytesAndMessagesCount>();
```

Comment:
nit: you can use '<>' instead of `<String, BytesAndMessagesCount>`

Reply:
Thanks; fixed in iteration f925d4d